Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -92,8 +97,19 @@ private void resetYVelocity()
     
     private void applyGravity()
     {
-        CharacterController char_controller = _player.GetComponent<CharacterController>();
         _velocity.y += -45.81f * Time.deltaTime * SpeedManager.playerMovementScaling;
-        char_controller.Move(_velocity * Time.deltaTime * SpeedManager.playerMovementScaling);
+        _controller.Move(_velocity * Time.deltaTime * SpeedManager.playerMovementScaling);
+    }
+
+    private void OnControllerColliderHit(ControllerColliderHit hit)
+    {
+        if (!isGrounded && hit.normal.y < wallJumpSlope)
```

Comment:
can you move `hit.normal.y < wallJumpSlope` into a variable with descriptive name so we know what this is doing ?